src/components/DailyReportFormUpdate/index.js

```diff
@@ -10,17 +10,12 @@ import CreateTaskForm from '../CreateTaskForm';
 import { DAILY_REPORTS_QUERY } from '../../containers/DailyReportPage/DailyReportContainer';
 
 const validate = ({ title, plan, tasks }) => ({
-  title:
-    !title || title.trim().length === 0
-      ? 'Title is required.'
-      : false,
-  plan:
-    !plan || plan.trim().length === 0
-      ? 'Plan is required.'
-      : false,
+  title: !title || title.trim().length === 0 ? 'Title is required.' : false,
+  plan: !plan || plan.trim().length === 0 ? 'Plan is required.' : false,
   tasks:
-    !tasks || tasks.map(task => {
-      if (task.projectId === '' || task.url === '' || task.logtime === '') {
+    !tasks ||
+    tasks.map(task => {
+      if (task.projectId === '' || task.url === '' || task.logtime === '' || task.title === '') {
```

Comment:
maybe this better if([task.projectId, task.url, task.logtime, task.title].includes(''))